source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -303,15 +303,24 @@ def wrapper(*args, **kwargs):
             if objects[1] != 200 or 'journeys' not in objects[0]:
                 return objects
             for journey in objects[0]['journeys']:
+                args = dict(request.args)
+                if 'region' in kwargs:
+                    args['region'] = kwargs['region']
                 if "sections" not in journey:#this mean it's an isochrone...
-                    args = dict(request.args)
                     if 'to' not in args:
                         args['to'] = journey['to']['id']
                     if 'from' not in args:
                         args['from'] = journey['from']['id']
-                    if 'region' in kwargs:
-                        args['region'] = kwargs['region']
                     journey['links'] = [create_external_link('v1.journeys', rel='journeys', **args)]
+                else:
+                    args['min_nb_journeys'] = 5
+                    ids = {o['stop_point']['stop_area']['id']
+                           for s in journey['sections'] if 'from' in s
+                           for o in (s['from'], s['to']) if 'stop_point' in o}
+                    if ids:
+                        for id in ids:
+                            args.setdefault('allowed_id[]', []).append(id)
+                        journey['links'] = [create_external_link('v1.journeys', rel='other_schedules', **args)]
```

Comment:
I'm not convinced by `other_schedule` but I don't have a better thing to propose :wink: